### PR TITLE
Add tests targets to the pipelines variables file

### PR DESCRIPTION
**Pull request by vsebe**
Move hard-coded test targets to the variables file.
Reformat Jenkins files to default tests targets to the value set in the
variables file when no build parameter is provided.

[ci skip]

Signed-off-by: Violeta Sebe <vsebe@ca.ibm.com>

### CODE REVIEW

**Comment by vsebe** [comment]
[personal build](https://ci.eclipse.org/openj9/job/vs_Pipeline_Build_Test_JDK11_ppc64le_linux/6/console)

**Comment by vsebe** [comment]
@AdamBrousseau please review, thank you!

**Comment by vsebe** [comment]
fixed merge conflict

**Comment by vsebe** [comment]
[Pipeline_Build_Test_JDK11_ppc64le_linux](https://ci.eclipse.org/openj9/view/Sandbox/job/vs_Pipeline_Build_Test_JDK11_ppc64le_linux/7/console)
[Pipeline_Build_Test_JDK11_x86-64_linux_cm](https://ci.eclipse.org/openj9/view/Sandbox/job/vs_Pipeline_Build_Test_JDK11_x86-64_linux_cm/2/console)

**Comment by vsebe** [comment]
@AdamBrousseau - ready for review

**Comment by AdamBrousseau** [comment]
jenkins test sanity plinux jdk11

File: buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All [hunk 1]
```diff
@@ -324,6 +324,11 @@ def get_summary_table(identifier) {
         return ''
     }
 
+    if (!TESTS_TARGETS) {
```
Comment (AdamBrousseau) [inline buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All]: I'm confused why this is here if we set it earlier?

File: buildenv/jenkins/variables/defaults.yml [hunk 2]
```diff
@@ -90,6 +90,19 @@ build_discarder:
 restart_timeout:
   time: '5'
   units: 'HOURS'
+tests_targets:
```
Comment (AdamBrousseau) [inline buildenv/jenkins/variables/defaults.yml]: I'm not sure how this works. Lets sit together tomorrow.

Comment (vsebe) [reply buildenv/jenkins/variables/defaults.yml]: Those are maps with null values 
see https://learnxinyminutes.com/docs/yaml/

File: buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All (hunk 1 above)
Comment (vsebe) [reply buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All]: TESTS_TARGETS is set only for pipelines not for the wrappers.

**Comment by vsebe** [comment]
jenkins test sanity plinux jdk11

**Comment by vsebe** [comment]
@pshipton ready for your review

**Pull request merged**